models/staging/edfi_3/base/base_ef3__survey_section_responses.sql

```diff
@@ -0,0 +1,23 @@
+with section_responses as (
+    {{ source_edfi3('section_responses') }}
+),
+renamed as (
+    select 
+        tenant_code,
+        api_year,
+        pull_timestamp,
+        last_modified_timestamp,
+        file_row_number,
+        filename,
+        is_deleted,
+
+        v:id::string                                               as record_guid,
+        v:surveyResponseReference:surveyResponseIdentifier::string as survey_response_id,
+        v:surveySectionReference:surveyIdentifier::string          as survey_id,
+        v:sectionRating::decimal(9,3)                              as section_rating,
```

Comment:
Is there a reason to use float instead of decimal(9,3) when the ed-fi documentation specifically mentions decimal(9,3) ?  https://schema.ed-fi.org/datahandbook-v510/index.html#/SectionRating

Reply:
Yes, durability. One out of range number fails the whole build with a strict decimal, and this code runs across hundreds of ODSes and many versions and customizations. I can't guarantee that any given decimal range in the documentation is accurate, stable across versions, and not modified in any implementation.